app/src/myanmar/java/org/eyeseetea/malariacare/strategies/BaseActivityStrategy.java

```diff
@@ -83,4 +88,27 @@ public void onLogoutError(String message) {
             }
         });
     }
+
+    @Override
+    public void onBackPressed() {
+
+    }
+
+    @Override
+    public void onWindowFocusChanged(boolean hasFocus) {
+
+    }
+
+    @Override
+    public void goSettings() {
```

Comment:
This is done different i one of this 4 times you are saying, I thing it's better to do this in the ABaseActivityStrategy and change it in BaseActivityStrategy of eReferrals.